src/codegen/record.rs

```diff
@@ -114,6 +114,7 @@ pub fn generate(w: &mut dyn Write, env: &Env, analysis: &analysis::record::Info)
         &analysis.functions,
         &analysis.specials,
         None,
+        None, // TODO
```

Comment:
What's to do here?

Reply:
Oh, pass down `analysis.version` :grimacing: , good catch!